lib/vast/Conversion/FromHL/Passes.cpp

```diff
@@ -42,6 +42,10 @@ namespace vast::conv::pipeline {
         return pass(createVarsToCellsPass);
     }
 
+    pipeline_step_ptr evict_static_locals() {
+        return pass(createEvictStaticLocalsPass);
```

Comment:
add `.depends_on(refs_to_ssa)`?

Reply:
I don't think it depends on `refs_to_ssa`. I believe these passes can be run in any order, as the legalizers should make sure that the pass does not act on the wrong Variable declarations